packages/salesforcedx-vscode-replay-debugger/src/messages/i18n.ts

```diff
@@ -22,5 +22,7 @@ export const messages = {
   session_started_text: 'Apex Replay Debugger session started for log file %s.',
   session_terminated_text: 'Apex Replay Debugger session terminated.',
   no_log_file_text:
-    'The log file either is missing or does not have any log lines in it.'
+    'The log file either is missing or does not have any log lines in it.',
+  incorrect_log_levels_text:
+    'The log must be generated with log categories Apex code and Visualforce at the finest level.'
```

Comment:
@ruthemmanuelle , here's the error message to the user if their log levels do not meet requirements.